presentation/src/main/java/com/knocklock/presentation/home/HomeViewModel.kt

```diff
@@ -36,29 +39,50 @@ class HomeViewModel @Inject constructor(
         .stateIn(
             scope = viewModelScope,
             started = SharingStarted.WhileSubscribed(5_000),
-            initialValue = LockScreen()
+            initialValue = LockScreen(),
         )
 
     val homeScreenUiState = combine(
         homeMenuList,
         tmpHomeScreen,
-        savedHomeScreenState
+        savedHomeScreenState,
     ) { menuList, tmpScreen, savedScreen ->
         HomeScreenUiState.Success(
             menuList = menuList.toImmutableList(),
-            lockScreen = if (tmpScreen is TmpScreenState.Custom) tmpScreen.screen else savedScreen
+            lockScreen = if (tmpScreen is TmpScreenState.Custom) tmpScreen.screen else savedScreen,
         )
     }.stateIn(
         scope = viewModelScope,
         started = SharingStarted.WhileSubscribed(5_000),
-        initialValue = HomeScreenUiState.Loading
+        initialValue = HomeScreenUiState.Loading,
     )
 
-    fun saveLockScreen() {
+    fun saveLockScreen(context: Context) {
         viewModelScope.launch {
             val value = tmpHomeScreen.value
             if (value is TmpScreenState.Custom) {
-                saveLockScreenCase(value.screen)
+                val uri = Uri.parse((value.screen.background as LockScreenBackground.LocalImage).imageUri)
```

Comment:
```suggestion
                val uri = Uri.parse((value.screen.background as?LockScreenBackground.LocalImage).imageUri)
```

타입 캐스팅 안정성을 위해 as?를 사용하는게 좋겠어요! 항상 LocalImage 인 것이 보장이 안될 것 같아요 (추후 타입추가 고려)

Reply:
@jaeryo2357 반영할께영